aptos-move/framework/aptos-framework/sources/aptos_coin.spec.move

```diff
@@ -31,12 +31,13 @@ spec aptos_framework::aptos_coin {
     ///
     spec module {
         pragma verify = true;
-        pragma aborts_if_is_strict;
+        pragma aborts_if_is_partial;
```

Comment:
If this module level pragma `aborts_if_is_partial` leads to verification error, need to add them at the function level.